docs/deploy/python.rst

```diff
@@ -21,16 +21,187 @@ Verify Installation
 
+Get Started
+^^^^^^^^^^^
+After confirming that the package ``mlc_chat`` is installed, we can follow the steps
+below to chat with a MLC-compiled model in Python.
+
+First, let us make sure that the MLC-compiled ``model`` we want to chat with already exists.
+
+.. note::
+   ``model`` has the format ``f"{model_name}-{quantize_mode}"``. For instance, if
+   you used ``q4f16_1`` as the ``quantize_mode`` to compile ``Llama-2-7b-chat-hf``, you 
+   would have ``model`` being ``Llama-2-7b-chat-hf-q4f16_1``.
+
+If you do not have the MLC-compiled ``model`` ready:
+
+- Checkout :ref:`Try out MLC Chat<get_started>` to download prebuilt models for simplicity, or
+- Checkout :ref:`Compile Models via MLC<compile-models-via-MLC>` to compile models with ``mlc_llm`` (another package) yourself
+
+.. tabs ::
+
+   .. tab :: Check prebuilt models
+
+      If you downloaded prebuilt models from MLC LLM, by default:
+
+      - Model lib should be placed at ``./dist/prebuilt/lib/$(model)-$(arch).$(suffix)``.
+      - Model weights and chat config are located under ``./dist/prebuilt/mlc-chat-$(model)/``.
+
+      .. collapse:: Example
+
+         .. code:: shell
+
+            >>> ls -l ./dist/prebuilt/lib
+            Llama-2-7b-chat-hf-q4f16_1-metal.so  # Format: $(model)-$(arch).$(suffix)
+            Llama-2-7b-chat-hf-q4f16_1-vulkan.so
+            ...
+            >>> ls -l ./dist/prebuilt/mlc-chat-Llama-2-7b-chat-hf-q4f16_1  # Format: ./dist/prebuilt/mlc-chat-$(model)/
+            # chat config:
+            mlc-chat-config.json
+            # model weights:
+            ndarray-cache.json
+            params_shard_*.bin
+            ...
+
+   .. tab :: Check compiled models
+
+      If you have compiled models using MLC LLM, by default:
+
+      - Model libraries should be placed at ``./dist/$(model)/$(model)-$(arch).$(suffix)``.
+      - Model weights and chat config are located under ``./dist/$(model)/params/``.
+
+      .. collapse:: Example
+
+         .. code:: shell
+
+            >>> ls -l ./dist/Llama-2-7b-chat-hf-q4f16_1/ # Format: ./dist/$(model)/
+            Llama-2-7b-chat-hf-q4f16_1-metal.so  # Format: $(model)-$(arch).$(suffix)
+            ...
+            >>> ls -l ./dist/Llama-2-7b-chat-hf-q4f16_1/params  # Format: ``./dist/$(model)/params/``
+            # chat config:
+            mlc-chat-config.json
+            # model weights:
+            ndarray-cache.json
+            params_shard_*.bin
+            ...
+
+After making sure that the files exist, from the ``mlc-llm`` directory, we can
+run the following code with the conda environment you used to install ``mlc_chat``:
+
+.. code:: python
+
+   from mlc_chat.chat_module  import ChatModule
+   cm = ChatModule(model='Llama-2-7b-chat-hf-q4f16_1')  # Create a ChatModule instance
+   cm.generate(prompt="What is the meaning of life?") # Generate a response for a given prompt
+   runtime_stats = cm.runtime_stats_text()  # Print some runtime statistics for the generation
```

Comment:
`cm.runtime_stats_text()` was renamed to `cm.stats()` in #688. The wheel build are in progress and will be ready shortly.